CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ cmake_policy(SET CMP0025 NEW)
 include(CMakeDependentOption)
 include(CMakePackageConfigHelpers)
 
-project(Chai LANGUAGES C CXX VERSION 2024.02.1)
+project(Chai LANGUAGES C CXX VERSION 2024.02.2)
```

Comment:
new version should be 2024.07.0

Reply:
I'm doing a 2024.02.2 release first so that there is a version of CHAI that works with RAJA 2024.02.2.